package.json

```diff
@@ -50,11 +50,15 @@
 		"micromatch": "^4.0.2",
 		"ms": "^2.1.3",
 		"pg": "^8.5.1",
+		"pino": "^6.11.1",
+		"pino-pretty": "^4.5.0",
+		"pino-tee": "^0.3.0",
 		"snake-case": "^3.0.4",
```

Comment:
**pino-pretty** should either be:
* Only active in development (and also put in devDependencies)
* Removed completely